librz/hash/hash.c

```diff
@@ -479,9 +513,11 @@ RZ_API RZ_OWN char *rz_hash_cfg_get_result_string(RZ_NONNULL RzHashCfg *md, RZ_N
 	HashCfgConfig *mdc = (HashCfgConfig *)rz_list_iter_get_data(it);
 	rz_return_val_if_fail(mdc, NULL);
 
-	if (!strncmp(name, "entropy", 7)) {
+	if (!strncmp(name, "entropy", strlen("entropy"))) {
```

Comment:
You could use `rz_str_startswith()` instead

Reply:
is not optimized.